real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -26,6 +26,23 @@
 	// "C:\\Users\\abc\\buggy-log\\VisionSystem.exe";
 
 	public static String LOG_FILE_LOCATION = "logs";
+	public static String LOG_STOP_MESSAGE = "STOP_LOGGING";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1273020439/issues/source?bid=2938665&fileBranchId=2938743#l29)